diff --git a/modules/shared/home-manager/default.nix b/modules/shared/home-manager/default.nix
index 8290452..139b171 100644
--- a/modules/shared/home-manager/default.nix
+++ b/modules/shared/home-manager/default.nix
@@ -21,7 +21,15 @@
     fd
     findutils
     ffmpeg
-    yt-dlp
+     (pkgs.yt-dlp.overridePythonAttrs (old: {
+       version = "2025.12.08";
+       src = pkgs.fetchFromGitHub {
+         owner = "yt-dlp";
+         repo = "yt-dlp";
+         rev = "2025.12.08";
+         sha256 = "1s4gbf49y6mpnfqy7n0lz2klfs9q1cbzcrg9mp353bl2zw68qknb";
+       };
+     }))
     glab
     autossh
 
